tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_managers/raster_dt_process_manager.h

```diff
@@ -0,0 +1,118 @@
+/**
+ * @file raster_dt_process_manager.h
```

Comment:
Not a big issue, but when I think of dt I initially think of delta t. Did you have any other names in mind?

Reply:
Yea, it suppose to relate to dual transitions, but not sure what would be an alternative abbreviation for this. Open to ideas.

Reply:
@mpowelson any ideas or leave it for now?